about.html

```diff
@@ -0,0 +1,153 @@
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script src="https://kit.fontawesome.com/870e4168d3.js" crossorigin="anonymous"></script>
+    <script
+      src="https://kit.fontawesome.com/4f02d055e5.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css"/>
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  </head>
+
+  <body>
+    <header>
+      <div class="top-bar">
+        <ul class="top-bar-ul">
+          <li><i class="fab fa-facebook-f"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li>English</li>
+          <li>Mypage</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="header-bar">
+        <div class="logo">
+          <a href="./index.html"
+            ><div class="logo-climate">
+            <p>SAVE THE PLANET </p>
+          </div></a
+          >
+          <div class="closeBtn"><i class="fas fa-times fa-2x"></i></div>
+        </div>
+        <nav>
+          <ul class="nav-ul">
+            <li><a class="menu-link" href="./index.html">Home</a></li>
+            <li><a class="menu-link" href="#program">Program</a></li>
+            <li><a class="menu-link" href="#">Join</a></li>
+            <li><a class="menu-link" href="#partner">Sponsors</a></li>
+            <li><a class="menu-link" href="#">News</a></li>
+          </ul>
+        </nav>
+        <button class="campaign">Campaign</button>
+      </div>
+    </header>
+    <main>
+      <section class="info">
+        <div class="menu">
+          <i class="fas fa-bars fa-2x"></i>
+        </div>
+        <h2>"LET U ALL HELP SAVE OUR CLIMATE"</h2>
+        <h1>GLOBAL CLIMATE FIXING CONFERENCE 2022</h1>
+        <hr />
+        <div class="info-descr">
+          <p>
+            This is the official page of the World Climate Fixing Conference
+            which is held annually.This conference is held tech company that 
+            are solving the world problems of climate condition and change.
+          </p>
+        </div>
+        <div class="info-contact">
+          <h3>
+            Please contact us per Email for any further questions about
+             global climate fixing conference 2022.
+          </h3>
+          <h4>globalclimateconference@org</h4>
+        </div>
+      </section>
+      <section class="the-logo">
+        <h2 class="section-title">
+          Creative Commons<br />Global Climate Fixing Conference 2022
+        </h2>
+        <hr />
+        <p>
+          This year conference will bring together investors, experts, and technocrat from every
+          field of the software industry and climate fixing companies and other startups.
+        </p>
+        <div class="logo-climate">
+          <div class="logo-climate">
+            <p>SAVE THE PLANET</p>
+          </div>
```

Comment:
- [ ] Please center this logo and add a border around it as seen in the original design

![image](https://user-images.githubusercontent.com/4247740/159021034-f0f5a3d6-d431-463e-afdf-bd12d993b0f3.png)

Reply:
- [ ] As requested by the previous reviewer,  kindly center the logo and add a border around it as seen in the original design.
<img width="949" alt="image" src="https://user-images.githubusercontent.com/77190203/159052503-9107b9ef-8dee-424a-b26d-e4d518bee831.png">